packages/in_app_purchase/in_app_purchase_android/example/lib/main.dart

```diff
@@ -291,8 +289,7 @@ class _MyAppState extends State<_MyApp> {
                                   : null);
                       if (productDetails.id == _kConsumableId) {
                         _inAppPurchasePlatform.buyConsumable(
-                            purchaseParam: purchaseParam,
-                            autoConsume: _kAutoConsume || Platform.isIOS);
+                            purchaseParam: purchaseParam);
```

Comment:
same as above

Reply:
fixed in [38eaffa](https://github.com/flutter/plugins/pull/6163/commits/38eaffa10e34d442ad98669b697221ea513615a4)